drivers/crypto/Kconfig.intel

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+config CRYPTO_INTEL_SHA
```

Comment:
Is it a generic "intel-wide" SHA? At least a comment about where the device can be found (CAVS/ACE) or a device type etc.

Reply:
Specified ACE and CAVS in "help" part of the config.